packages/shared/src/hooks/useFeedLayout.ts

```diff
@@ -1,45 +1,43 @@
-import { useMemo } from 'react';
-import { SharedFeedPage } from '../components/utilities';
-import { AllFeedPages } from '../lib/query';
-import { useActiveFeedNameContext } from '../contexts/ActiveFeedNameContext';
 import { useViewSize, ViewSize } from './useViewSize';
-
-interface UseFeedLayoutProps {
-  feedName?: AllFeedPages;
-  feedRelated?: boolean;
-}
+import { useActiveFeedNameContext } from '../contexts/ActiveFeedNameContext';
+import { OtherFeedPage } from '../lib/query';
+import { FeedPage, FeedPageLayoutMobile } from '../components/utilities';
 
 interface UseFeedLayout {
   shouldUseMobileFeedLayout: boolean;
+  FeedPageLayoutComponent: React.ComponentType;
+}
+
+export enum FeedPagesWithMobileLayout {
+  MyFeed = 'my-feed',
+  Popular = 'popular',
+  Search = 'search',
+  Upvoted = 'upvoted',
+  Discussed = 'discussed',
+  Bookmarks = 'bookmarks',
+  UserPosts = 'user-posts',
+  UserUpvoted = 'user-upvoted',
```

Comment:
We already have these strings defined somewhere. Can we use values from the other enums? E.g. for MyFeed, we already define it here AFAIK: https://github.com/dailydotdev/apps/blob/main/packages/shared/src/components/utilities/common.tsx#L163